src/browser/api_protocol/api_handlers/frame.ts

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2017 OpenFin Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import * as apiProtocolBase from './api_protocol_base';
+import { Frame } from '../../api/frame';
+import { Identity } from '../../../shapes';
+import { ActionSpecMap } from '../shapes';
+
+const successAck: object = {
+    success: true
+};
+
+export class FrameApiHandler {
```

Comment:
Some other API handlers has migrated (forward) to _functional_ style, e.g. _window api handler_ (`init` function)

Reply:
This one I'd rather leave for now, it is a class, so just the `new` feels natural enough for now and I'd rather not refactor atm